Dockerfile

```diff
@@ -2,22 +2,12 @@
 FROM runatlantis/atlantis:v0.17.2
 
 # Install required packages
-RUN apk --update --no-cache add ca-certificates openssl openssh-client curl git
-
-# The jq package provided by alpine:3.13 (jq 1.6-rc1) is flagged as a 
-# high severity vulnerability, so we install the latest release ourselves
-# Reference: https://nvd.nist.gov/vuln/detail/CVE-2016-4074 (this is present on jq-1.6-rc1 as well)
-RUN \
-    # Install jq-1.6 (final release)
-    curl -s -L -o /tmp/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && \
-    mv /tmp/jq /usr/local/bin/jq && \
-    chmod +x /usr/local/bin/jq
+RUN apk --update --no-cache add ca-certificates openssl openssh-client curl git nodejs nodejs-npm
```

Comment:
it might be good to add `jq` to the list of packages in case users want to use it to parse/lookup values from the Infracost JSON?

Reply:
Added